iocage/__init__.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/local/bin/python3.6
+# -*- coding: utf-8 -*-
+
+import re
+import sys
+import os
+
+__dirname = os.path.dirname(__file__)
+iocage_lib_dir = os.path.join(__dirname, "lib")
+sys.path = [iocage_lib_dir] + sys.path
```

Comment:
I agree this is a problem, that needs to be addressed. Maybe you can help me find another solution for the problem:

The interface of the library was meant to be accessible like this

```
import iocage
jail = iocage.Jail()
```

That trick was done with the iocage/__init__.py file that wraps the publicly exposed classes. I didn't find a way to import classes from a file with the same name without repeating the name (e.g. `iocage.lib.Jail.Jail`), but that's another issue.

I will look into this a little later, but this is a blocker and needs to be addressed!